docs/video/01-build/06-Develop-Handle-Incoming-Video-Call.mdx

```diff
@@ -28,28 +26,30 @@ import { ABlock } from "@site/src/css/SharedStyling";
 Earlier, we set up the Push Video Stream and event listeners in the [Stream API](../stream-video) section. Now, when there is an incoming video call request, the `stream` emits a `RequestVideo` event. This event can be used to display a UI modal to the user, allowing them to accept or deny the incoming call.
 
 ```typescript
-stream.on(CONSTANTS.STREAM.VIDEO, async (data) => {
+stream.on(CONSTANTS.STREAM.VIDEO, async (data: TYPES.VIDEO.EVENT) => {
   if (data.event === CONSTANTS.VIDEO.EVENT.REQUEST) {
     // RequestVideo Event is fired when there is an incoming video call request
-    // store this data.peerInfo which will be used to accept/reject the call
-    peerInfo = data.peerInfo;
+    // store the incoming peer address which will be used to accept/reject the call
+    let incomingCallerAddress = data.peerInfo.address;
```

Comment:
refer to chat to ensure parameters and api are segregated, also storing peerInfo.address can come here but with comments that it's recommend to store the caller address to make the interface better